src/build.d

```diff
@@ -914,7 +927,8 @@ void processEnvironment()
     else
         env.getDefault("ZIP", "zip");
 
-    env.getDefault("ENABLE_WARNINGS", "0");
+    // TODO: this isn't being used for anything yet...
+    env.getNumberedBool("ENABLE_WARNINGS");
```

Comment:
This exlcludes `ENABLE_WARNINGS` from the environment instead of setting it to `0` by default.
(We should make `ENABLE_WARNINGS` imply `-w` in another PR)

Reply:
`getDefault` doesn't set environment variables, it sets values in the global `env` variable.  These `ENABLE_*` variables are only referenced once and now we don't need to set them because we are interpreting empty as `false`.

Reply:
Okay (I meant `env` not `environment` but fine anyway)

Reply:
I ended up just having `getNumberedBool` forward to `getDefault` so it would check the environment variables first anyway.